app/views/users/_users_list.html.haml

```diff
@@ -0,0 +1,15 @@
+#users-list
+  %table.table
+    %thead.panel-heading
+      %tr
+        %th= sort_link(q, :email, t('.email'), { default_order: { email: :desc, email: :asc } }, { remote: true, id: 'order-email' } )
```

Comment:
Please don't add attributes solely for testing convenience unless necessary (which this is not).